packages/next/src/server/config-shared.ts

```diff
@@ -195,6 +195,10 @@ export interface ReactCompilerOptions {
 }
 
 export interface ExperimentalConfig {
+  /**
+   * A target memory limit for turbo, in bytes.
+   */
+  turboMemoryLimit?: number
```

Comment:
This needs to be added to the config-schema as well https://github.com/vercel/next.js/blob/canary/packages/next/src/server/config-schema.ts and we should set a default value (even if `undefined`) in this file.